packages/react-native-reanimated/package.json

```
@@ -93,7 +93,7 @@
     "@babel/plugin-transform-unicode-regex": "^7.0.0-0",
     "@babel/preset-typescript": "^7.16.7",
     "convert-source-map": "^2.0.0",
-    "react-native-is-edge-to-edge": "1.1.6"
+    "react-native-is-edge-to-edge": "^1.1.7"
```

```suggestion
    "react-native-is-edge-to-edge": "1.1.7"
```
Can we point to a specific version instead?

---

Done!